identity/identity.go

```diff
@@ -287,3 +307,11 @@ func (i *Identity) ValidateNID() error {
 
 	return nil
 }
+
+func (i *Identity) GetCredential(ctx context.Context, config sqlxx.JSONRawMessage) (*CredentialsConfig, error) {
```

Comment:
Do we need this new method? If not, let's remove it as it's just JSON decoding :)